### PR TITLE
Add CI checks for file formatting and style checks

### PR DESCRIPTION
**Pull request by aaronfranke**
These scripts were copied from the main repository and modified.

GitHub Actions won't run until after the PR is merged, since we are adding the checks for the first time.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -82,19 +86,13 @@ def get_subscriptions(self):
         self.send(subscriptions_msg)
 
     def subscribe(self, channel, channel_id):
-        subscribe_msg = {
```
Comment (hpvb) [inline bot.py]: Is there any way to not change the formatting on these messages?

File: bot.py [hunk 2]
```diff
@@ -82,19 +86,13 @@ def get_subscriptions(self):
...
-            ]
-        }
+        subscribe_msg = {"msg": "sub", "id": channel, "name": "stream-room-messages", "params": [channel_id, False]}
         self.send(subscribe_msg)
 
     def format_issue(self, repository, issue):
-        headers = { 'User-Agent': 'Godot Issuebot by hpvb', }
+        headers = {
+            "User-Agent": "Godot Issuebot by hpvb",
+        }
```
Comment (hpvb) [inline bot.py]: I'm also not sure why this ends up on two lines when larger structures end up on a single line.

**Review by hpvb — CHANGES_REQUESTED** [review]
I'm OK with applying some kind of code style to this repository! But I'd really like to see those two things changed.

File: bot.py (hunk 1 above)
Comment (aaronfranke) [reply bot.py]: ~~No, [black "is not configurable"](https://black.readthedocs.io/en/stable/the_black_code_style.html).~~ EDIT: The trailing commas control this.

File: bot.py (hunk 2 above)
Comment (aaronfranke) [reply bot.py]: This one is because of the trailing comma. If we remove the trailing comma, it will become one line again.

Comment (hpvb) [reply bot.py]: Would you mind making those changes then so I don't have to merge this large diff and then create another large one right away to correct this?

Comment (aaronfranke) [reply bot.py]: Done

**Review by akien-mga — APPROVED** [review]
@hpvb's review comments have been addressed so this should be good to go.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**